packages/eslint-config/index.js

```diff
@@ -34,6 +35,13 @@ module.exports = {
         ],
         "import/no-unresolved": [
             "warn"
-        ]
+        ],
+        "no-secrets/no-secrets": [
+            "warn",
+            { "additionalRegexes": { 
+                "Basic Auth": "Authorization: Basic [A-Za-z0-9+/=]*",
+                "Common Pattern": "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-~|]).{8,}$"
```

Comment:
not blocking this PR, it may be better if we can have a document (maybe a readme in eslint-config folder?) talking about the common secret pattern (as well as other lint rules). It may be hard for others to understand the common pattern.